tests/jsanalyze_test.js

```diff
@@ -6,6 +6,23 @@ const fs = require('fs-extra');
 const should = require('should'); // eslint-disable-line no-unused-vars
 const jsanalyze = require('../lib/jsanalyze');
 
+function sortObject (o) {
```

Comment:
No need to duplicate code. Export `sortObject()` from `jsanalyze.js` and call `jsanalyze.sortObject()`.

Reply:
:+1: fixed